src/__tests__/ParseUser-test.js

```diff
@@ -1009,4 +1010,26 @@ describe('ParseUser', () => {
     const authProvider = user.linkWith.mock.calls[0][0];
     expect(authProvider).toBe('testProvider');
   });
+
+  it('can get the current user even encrypted', async () => {
+    CoreManager.set('ENCRYPTED_USER', true);
+    CoreManager.set('ENCRYPTED_KEY', 'secret');
```

Comment:
I see encrypted key is defaulted to null. Can you write a test when encrypted user is true but you don’t set the encrypted key?

Reply:
Done it, also in the conditional to encrypt the data I check if it's null.